internal/config/helpers.go

```diff
@@ -58,6 +58,14 @@ func InNSList(nn []interface{}, ns string) bool {
 func MustK9sUser() string {
 	usr, err := user.Current()
 	if err != nil {
+		envUsr := os.Getenv("USER")
+		if envUsr != "" {
+			return envUsr
+		}
+		envUsr = os.Getenv("LOGNAME")
```

Comment:
I vaguely recall that LOGNAME is in some standard, unlike USER, so perhaps these should be on the opposite order...

Reply:
@tomaszkiewicz @porridge You are correct LOGNAME was the gran papa. 
USER was introduced in bsd (i think?).
If both are set they should yield the same value tho...

Reply:
So do you expect any changes here or can we leave it as it is?

Reply:
Not sure whether you're asking me or @derailed ... I'm fine with either way.